src/components/row/row.component.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import StyledRow from "./row.style";
+
+const Row = ({
+  columns,
+  children,
+  columnClasses,
+  columnDivide,
+  gutter,
+  className,
+}) => {
+  const columnsCount = columns || React.Children.toArray(children).length;
```

Comment:
There's a `React.Children.count(children)` that could be used here instead

Reply:
It looks that it has to stay that way because there is an unit test that checks if passing `null` as a children counts it toward the column count. Looks like using `React.Children.count` actually counts the `null`, while `React.Children.toArray(children).length` does not (as intended).